src-ghc/Pact/PersistPactDb/Regression.hs

```diff
@@ -41,10 +41,10 @@ runRegression p = do
   let ks = KeySet [PublicKey "skdjhfskj"] (Name "predfun" def)
   _writeRow pactdb Write KeySets "ks1" ks v
   assertEquals' "keyset write" (Just ks) $ _readRow pactdb KeySets "ks1" v
-  let mod' = Module "mod1" "mod-admin-keyset" Nothing "code" (H.hash "code") mempty
+  let mod' = Module "mod1" "mod-admin-keyset" (DocModel Nothing M.empty) "code" (H.hash "code") mempty
   _writeRow pactdb Write Modules "mod1" mod' v
   assertEquals' "module write" (Just mod') $ _readRow pactdb Modules "mod1" v
-  assertEquals' "hash of commit 3" (-3314666147157983816) (hashWithSalt 1 <$> commit v)
+  assertEquals' "hash of commit 3" 5222047446822369474 (hashWithSalt 1 <$> commit v)
```

Comment:
This is the one detail I'm not so sure about. Changing the internal format for (meta in) terms changes the resulting hash. We could keep the old hash but we'd have to translate to the old format to do so.

Reply:
pinging @slpopejoy